README.md

```diff
@@ -114,6 +114,10 @@ custom:
         description: 'Http endpoint'
         config:
           endpoint: # required # "https://{DOMAIN}/{PATH}"
+    replacements: # allows to pass variables from here to velocity templates
```

Comment:
Do you mean `substitutions` instead of `replacements`?

Reply:
fixed that, thanks